common4j/src/main/com/microsoft/identity/common/java/cache/AbstractAccountCredentialCache.java

```diff
@@ -151,7 +151,8 @@ protected List<Credential> getCredentialsFilteredByInternal(@NonNull final List<
                                                                 @Nullable final String target,
                                                                 @Nullable final String authScheme,
                                                                 @Nullable final String requestedClaims,
-                                                                @Nullable final String kid) {
+                                                                @Nullable final String kid,
+                                                                @Nullable Boolean mustMatchExactClaims) {
```

Comment:
>@Nullable Boolean [](http://example.com/codeflow?start=64&length=17)

I'm looking at this and the new line on 170 -- should this just be a primitive lowercase `boolean`?

Reply:
done in latest iteration